app/views/protocols/form/study_form_sections/_study_information.html.haml

```diff
@@ -27,6 +27,13 @@
       = t(:protocols)[:studies][:information][:subtext]
   .panel-body.container-fluid
     .form-group.row
+      = form.label :research_master_id, class: 'col-lg-2 control-label rm-id' do
```

Comment:
Needs to be replicated in "app/views/dashboard/protocols/form/study_form_sections/_study_information.html.haml"